Patch-Source: https://sources.debian.org/data/main/libg/libgit-raw-perl/0.90+ds-2/debian/patches/libgit2_1.7.1
--
Description: adjust tests to libgit2 1.7.1
 * t/08-branch.t: relog handling changed, adjust to the new 2 entries
 * t/19-push.t: the pack_progress callback returns all kind of values
   (timing issue?), mark as TODO
Origin: vendor
Bug-Debian: https://bugs.debian.org/1057318
Forwarded: not-needed (upstream uses a vendored libgit2)
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2023-12-09

--- a/t/08-branch.t
+++ b/t/08-branch.t
@@ -51,7 +51,7 @@
 
 my $reflog = $look -> reflog;
 my @entries = $reflog -> entries;
-is scalar(@entries), 1;
+is scalar(@entries), 2;
 
 ok (!defined (Git::Raw::Branch -> lookup($repo, 'nonexistent_local', 1)));
 ok (!defined (Git::Raw::Branch -> lookup($repo, 'nonexistent_remote', 0)));
--- a/t/19-push.t
+++ b/t/19-push.t
@@ -31,7 +31,10 @@
 					$stage == Git::Raw::Packbuilder->DELTAFICATION);
 				if ($stage == Git::Raw::Packbuilder->ADDING_OBJECTS)
 				{
+					TODO: {
+					local $TODO = 'flaky with libgit2 1.7.1';
 					is $current, 1;
+					}
 					is $total, 0;
 				}
 				else
@@ -70,7 +73,10 @@
 				ok ($stage == 0 || $stage == 1);
 				if ($stage == 0)
 				{
+					TODO: {
+					local $TODO = 'flaky with libgit2 1.7.1';
 					is $current, 1;
+					}
 					is $total, 0;
 				}
 				else
